src/user/redux/modules/change_password.js

```diff
@@ -1,12 +1,20 @@
-import { FETCHED_MEMBER } from '../../../shared/redux/modules/member.js'
+import { createAction } from 'redux-actions'
```

Comment:
That sounds good, not sure I grasp it fully though. Will have to do some background reading.